store/action-menu.js

```diff
@@ -73,9 +73,13 @@ export const mutations = {
   },
 
   togglePromptRemove(state, resources = []) {
-    state.showPromptRemove = !state.showPromptRemove;
-    if (!isArray(resources)) {
-      resources = [resources];
+    if (!resources.length) {
```

Comment:
If resources is a single item, resources.length is null, since its not an array, so it goes through the wrong path and resources is not converted to an array.


This worked for me:

```

T  togglePromptRemove(state, resources) {
    if (!resources) {
      state.showPromptRemove = false;
      resources = [];
    } else {
      state.showPromptRemove = !state.showPromptRemove;
      if (!isArray(resources)) {
        resources = [resources];
      }
    }

    state.toRemove = resources;
  },

```